api-tests/platform/targets/tgt_dev_apis_stdc/target.cmake

```diff
@@ -15,6 +15,61 @@
 # * limitations under the License.
 #**/
 
+###############################################################################
+# FUNCTION: _create_psa_stdc_exe()
+#  Function for generating PSA stdc test binaries linking with libraries
+#  external to the psa-arch-tests project. This function requires
+#  PSA_STORAGE_LIB_FILENAME to be specificed on the cmake command line,
+#  where the symbol is defined as the full path to the external PSA storage
+#  library to test. e.g.
+#    cmake ... -DPSA_STORAGE_LIB_FILENAME=/wdir/usr/lib/libpsastorage.so
+#  If the function is being used to generate a test binary for testing 
+#  the mbed-crypto library then the function requires 
+#  PSA_CRYPTO_LIB_FILENAME to be specificed on the cmake command line,
+#  where the symbol is defined as the full path to the external PSA crypto
+#  library to test. e.g.
+#    cmake ... -DPSA_CRYPTO_LIB_FILENAME=/wdir/mbed-crypto/library/    \
+#                                                               libmbedcrypto.a
+# ARGUMENTS:
+#   _exe_name     Name of the test binary to generate.
+#   _api_dir      PSA API directory name e.g. crypto, 
+#                 internal_trusted_storage or protected_storage.
+###############################################################################
+function(_create_psa_stdc_exe _exe_name _api_dir)
+
+	# Create the PSA test binary.
+	set(EXE_NAME ${_exe_name})
+
+	# Define PSA_STORAGE_LIB_NAME to be the name of the PSA Storage library to be tested.
```

Comment:
You may want to guard this code for Crypto suite as this is related to storage tests.

Reply:
Guard added as requested, which I think is optimal because psa storage and psa crypto can be built alone, or they can also be tested in combination.